### PR TITLE
in_forward: Add descrioptions for secure forward related parameters

### PR DESCRIPTION
**Pull request by cosmo0920**
To work as secure forward connection, there are a few parameters to be needed to specify:

* shared_key
* self_hostname
* security.users

Corresponding to https://github.com/fluent/fluent-bit/pull/8665.

### CODE REVIEW

File: pipeline/inputs/forward.md [hunk 1]
```diff
@@ -17,6 +17,9 @@ The plugin supports the following configuration parameters:
 | Buffer\_Chunk\_Size | By default the buffer to store the incoming Forward messages, do not allocate the maximum memory allowed, instead it allocate memory when is required. The rounds of allocations are set by _Buffer\_Chunk\_Size_. The value must be according to the [Unit Size ](../../administration/configuring-fluent-bit/unit-sizes.md)specification. | 1024000 |
 | Tag_Prefix          | Prefix incoming tag with the defined value.|  |
 | Tag                 | Override the tag of the forwarded events with the defined value.|  |
+| Shared\_Key         | Shared key for secure forward authentication. |  |
+| Self\_Hostname      | Hostname for secure forward authentication.   |  |
+| Secutiry.Users      | Specify username and password pairs for secure forward authenticaiton. | |
```
Comment (lecaros) [inline pipeline/inputs/forward.md:22]:
```suggestion
| Security.Users      | Specify the username and password pairs for secure forward authentication. | |
```

Comment (lecaros) [reply pipeline/inputs/forward.md:22]: can you add an example of the Security.Users param?

**Review by lecaros — CHANGES_REQUESTED** [review]

File: pipeline/inputs/forward.md (hunk 1 above)
Comment (cosmo0920) [reply pipeline/inputs/forward.md]: Sure. I will do.

**Review by lecaros — APPROVED** [review]

**Comment by lecaros** [comment]
Thanks @cosmo0920! Merged.

**Pull request merged**